tests/test_audit.py

```
@@ -93,6 +98,77 @@ def test_generated_audit_target_runs_expected_tools(
+    [
+        ("audit-workflow-pure", "AuditWorkflowPure", "audit_workflow_pure", False),
+        ("audit-workflow-rust", "AuditWorkflowRust", "audit_workflow_rust", True),
+    ],
+)
+def test_generated_audit_workflow_has_expected_contract(
+    copier: CopierFixture,
+    tmp_path: Path,
+    target_dir: str,
+    project_name: str,
+    package_name: str,
+    use_rust: bool,
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)